systems/andes_user_guide.rst

```diff
@@ -47,8 +47,8 @@ Andes contains 704 compute nodes and 9 GPU nodes. Andes has two partitions:
 | Partition   | Node Count  | Memory  | GPU               | CPU                                |
 +=============+=============+=========+===================+====================================+
 | batch       | 704         | 256 GB  | N/A               | [2x] AMD EPYC 7302 16Core Processor|
-| (default)   |             |         |                   | 3.0 GHz, 32 SMT                    |   
-|             |             |         |                   | (total 32 cores, 64 SMT *per node*)|
+| (default)   |             |         |                   | 3.0 GHz, 16 SMT                    |   
```

Comment:
Should we just remove SMT now and stick with cores?

Reply:
Yes. Just updated that part